### PR TITLE
allow agnhost to set TLS gRPC

### PR DESCRIPTION
**Pull request by SergeyKanzhelev**
#### What type of PR is this?

/kind bug

#### What this PR does / why we need it:

- In order to test a fix for https://github.com/kubernetes/kubernetes/issues/119093 we need to have agnhost exposing gRPC TLS

#### Which issue(s) this PR fixes:

Step towards https://github.com/kubernetes/kubernetes/issues/119093

#### Does this PR introduce a user-facing change?
```release-note
NONE
```

### CODE REVIEW

File: test/images/agnhost/grpc-health-checking/grpc-health-checking.go [hunk 1]
```diff
@@ -49,13 +50,19 @@ var (
...
 
 func init() {
 	CmdGrpcHealthChecking.Flags().IntVar(&port, "port", 5000, "Port number.")
 	CmdGrpcHealthChecking.Flags().IntVar(&httpPort, "http-port", 8080, "Port number for the /make-serving and /make-not-serving.")
 	CmdGrpcHealthChecking.Flags().IntVar(&delayUnhealthySec, "delay-unhealthy-sec", -1, "Number of seconds to delay before start reporting NOT_SERVING, negative value indicates never.")
 	CmdGrpcHealthChecking.Flags().StringVar(&service, "service", "", "Service name to register the health check for.")
+	CmdGrpcHealthChecking.Flags().StringVar(&certFile, "tls-cert-file", "",
+		"File containing an x509 certificate for gRPC TLS. (CA cert, if any, concatenated after server cert).")
+	CmdGrpcHealthChecking.Flags().StringVar(&privKeyFile, "tls-private-key-file", "",
+		"File containing an x509 private key matching --tls-cert-file.")
```
Comment (bart0sh) [inline test/images/agnhost/grpc-health-checking/grpc-health-checking.go:65]: As these 2 parameters can only be used together we should probably validate that user specifies either both of them or none.

Comment (SergeyKanzhelev) [reply test/images/agnhost/grpc-health-checking/grpc-health-checking.go:65]: Do you mean report an error on startup? We only use them if both are specified indeed

Comment (SergeyKanzhelev) [reply test/images/agnhost/grpc-health-checking/grpc-health-checking.go:65]: Added

**Pull request merged**